common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```
@@ -140,6 +140,7 @@ cACTION_APMANAGER_CLIENT_DISALLOW_REQUEST:
   _type: class
   mac: sMacAddr
   bssid: sMacAddr
+  validity_period: uint16_t
```

(nitpick) can we add an explicit unit here?
```suggestion
  validity_period_sec: uint16_t
```